### DISCUSSION TITLE
请求返回code: -462是什么意思呢？是要登录网易云账号才能继续调用接口吗？

### DISCUSSION THREAD

**Discussion by Lsiyi1** [Binaryify/NeteaseCloudMusicApi#1737] [Q&A]
之前不用登录就能返回数据的接口，有一些现在好像都返回这个了，比如/song/detail歌曲详情的接口。
如果是因为要登录的话，请问有人知道登录怎么写吗？
'/login/cellphone?phone=xxx&password=xxx'用这个登录接口，我返回的还是-462。
![image](https://user-images.githubusercontent.com/132632834/236366101-affc074e-b313-46b1-a552-5cac4835c0fc.png)

**Comment by Lsiyi1** [comment]
好像是我下的服务器代码的问题，我用这里这个代码就可以调用了。

**Reply by single1231414** [reply]
请问你解决这个问题了吗

**Reply by single1231414** [reply]
我现在也是这个情况

**Reply by Lsiyi1** [reply]
> 请问你解决这个问题了吗
解决了，我之前不是下载的这里提供的接口代码的，然后我用了这里提供的代码就好了。如果你用的是这里下的代码，但是遇到了这个问题的话，不好意思，我不知道该怎么解决
